backend/api/CourseApi/getStudentsStats.js

```diff
@@ -0,0 +1,58 @@
+import knex from '~/db/knex';
+import auth from '~/middlewares/auth';
+
+const getStudentsStats = auth(async (request, response) => {
+
+  const authorId = request.body['authorId'];
+  const courseId = request.body['courseId'];
+  const students = await knex('coauthor').select().where({ courseId });
```

Comment:
Coauthors !== students, to find the students of the course do:

```
const students = await knex('user')
    .select('user.*')
    .join('courseUserIsLearning', { 'courseUserIsLearning.userId': 'user.id' })
    .where({ 'courseUserIsLearning.courseId': courseId, active: true });
```

Reply:
Coauthors automatically become students when you invite them.
Good to know this is confusing, I can see why. We might want a separate "invite students" button.

You cannot invite users, users have to find your course on their own and join it via the "Start Learning" button.

<img width="277" alt="image" src="https://user-images.githubusercontent.com/7578559/154615149-ca0304a8-ab9b-48aa-abb8-eeae0f6ed017.png">

Reply:
Done.